image.yaml

```diff
@@ -27,19 +27,19 @@ modules:
           - name: cct_module
             git:
                 url: https://github.com/jboss-openshift/cct_module.git
-                ref: 0.37.1
+                ref: 0.39.0
           - name: jboss-eap-7-image
             git:
                 url: https://github.com/jboss-container-images/jboss-eap-7-image.git
                 ref: EAP_725_CR2
       install:
-          - name: jboss.container.openjdk.jdk
-            version: '8'
+          - name: sso-jdk
```

Comment:
@hmlnarik Please specify module version for each of the `sso-jdk`, `eap-7.2-latest`, and `sso-rm-openjdk` modules (seems to be [required starting from CEKit v3.3.1](https://github.com/cekit/cekit/releases/tag/3.3.1))

Reply:
For EAP 7.2, this was copied from https://github.com/jboss-container-images/jboss-eap-7-image/blob/eap72/image.yaml#L22-L24. I'll adjust the sso ones.

Reply:
Ok, thanks